bifrost/renderer/+config.ts

```diff
@@ -1,7 +1,7 @@
 import { type ConfigNonHeaderFile } from "vite-plugin-ssr/types";
 
 export default {
-  passToClient: ["layoutProps", "pageProps", "redirectTo", "documentProps", "isLoggedIn", "scripts"],
+  passToClient: ["layoutProps", "pageProps", "redirectTo", "documentProps", "isLoggedIn", "scripts", "storyblokSlug", "storyblokStory"],
```

Comment:
i dont think the slug has to be passToClient? I think if you make storyblokSlug server-and-client below, it'll handle it.

Reply:
Yeah you're right it never actually gets passed back to client. I'm not sure if it needs to be `server-and-client`, I'll play around with it